cmd/vic-machine/create/create.go

```diff
@@ -326,21 +371,6 @@ func (c *Create) Flags() []cli.Flag {
 	return flags
 }
 
-func (c *Create) processVolumeStores() error {
```

Comment:
moved down below the main processParams to live with the other processX methods.